src/widget/OktaSignIn.js

```diff
@@ -130,7 +110,16 @@ var OktaSignIn = (function () {
         See: https://developer.okta.com/code/javascript/okta_sign-in_widget#cdn
       `);
 
-    var authClient = createAuthClient(options);
+    var authParams = _.extend({
```

Comment:
why not leave all those extra logic to `createAuthClient` function?

Reply:
The `createAuthClient` is not bound to a widget instance, so it doesn't have access to `this.options` (it is a factory function, also used by tests, when there is no widget instance). This block of code is copying specific properties from `this.options` onto `authParams`

Reply:
Basically, the old function took WIDGET options, the new function takes AUTHJS options.